libs/shared/contentful/src/lib/contentful.manager.spec.ts

```diff
@@ -52,4 +54,24 @@ describe('ContentfulManager', () => {
       expect(result.getOfferingForPlanId(planId)).toBeDefined();
     });
   });
+
+  describe('getServicesWithCapabilities', () => {
+    it('should return results', async () => {
+      mockClient.query = jest.fn().mockReturnValue({
+        data: {
+          serviceCollection: { items: [] },
+        },
+      });
+      const result = await manager.getServicesWithCapabilities();
+      expect(result).toBeDefined();
+      expect(result.serviceCollection.items).toHaveLength(0);
+    });
+
+    it('should return successfully with services and capabilities', async () => {
+      const queryData = ServicesWithCapabilitiesQueryFactory();
+      mockClient.query = jest.fn().mockResolvedValueOnce({ data: queryData });
+      const result = await manager.getServicesWithCapabilities();
+      expect(result).toBeDefined();
```

Comment:
Would be nice for this test to check the exact values of the response rather than just whether it's not undefined.